tests/test_project.py

```diff
@@ -45,3 +63,8 @@ def test_device_get_qos2(self):
         self.device.add_interface(interface_definition_unique)
         self.assertIs(self.device._get_qos(interface_definition_unique["interface_name"]), 2,
                       msg="qos should be '2' for 'unique' reliability")
+
+    def test_set_property(self):
```

Comment:
What is the purpose of this unit test?
`unset_property` is called. Should it not be `set_property`?